### PR TITLE
removed ggplot dependency

### PR DESCRIPTION
**Pull request by k-t-to**
plots updated to use base r

### CODE REVIEW

**Comment by k-t-to** [comment]
Sample Explorer > Summary Plots needs to be fixed

File: bin/functions/make_plots.R [hunk 1]
```diff
@@ -40,21 +43,25 @@ plot_pod_dist <- function(pod_df, pod_qs, in_dat, viewopt) {
...
+       xlab = x_ax_lab,
+       ylab = "Density",
+       xlim = x_ax_lims,
+       main = "Distribution of POD Estimates",
+       frame.plot = F)
+  box(lwd = 2, bty = "l")
+  polygon(d, col=rgb(1,0.75,0.15,0.5))
+  abline(v = pod_qs, lty = c("dotted", "solid", "dashed"), col = "blue", lwd = 2)
+  par(xpd = T)
+  legend("topright", legend = names(pod_qs), col = "blue", lty = c("dotted", "solid", "dashed"), title = "Quantiles", lwd = 2, inset = c(-0.15,0), cex = 0.75)
```
Comment (DataSciBurgoon) [inline bin/functions/make_plots.R]: Need to bring back the median and the credible intervals reporting either in the legend, as before, or someplace else. I don't see it in the code that was dropped, so I'm wondering if it was dropped in an earlier commit?

**Review by DataSciBurgoon — CHANGES_REQUESTED** [review]

**Comment by DataSciBurgoon** [comment]
When running this code the interface is not returning the POD or the credible interval. See image:
Screenshot 2020-12-28 134001.

Sample explorer works well.

The input data works well.

**Comment by DataSciBurgoon** [comment]
Looks like the PR is closed; did you want to merge the commits on redesign, or abandon?

**Comment by k-t-to** [comment]
Closed. [New PR](https://github.com/k-t-to/ToxPOD/pull/6) that will merge redesign branch into master

**Pull request closed without merging**